runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/client/commits/StreamingEngineWorkCommitter.java

```diff
@@ -86,16 +86,19 @@ public final class StreamingEngineWorkCommitter implements WorkCommitter {
   public static Builder builder() {
     return new AutoBuilder_StreamingEngineWorkCommitter_Builder()
         .setBackendWorkerToken(NO_BACKEND_WORKER_TOKEN)
+        .setWeigher(
+            WeightedSemaphore.create(
```

Comment:
maybe require this to be set by caller instead of hiding it? It would be nice for the callers to know about it because then they could display it on their status page

Reply:
done